app/helpers/verification_helper.rb

```diff
@@ -87,7 +87,7 @@ def is_unverified_evidences?(person)
       end
     end
 
-    aasm_states.include?('outstanding')
+    aasm_states.any? { |state| Eligibilities::Evidence::OUTSTANDING_STATES.include?(state) }
```

Comment:
Non-blocking comment -

how about using below. we can avoid looping this way.
`(Eligibilities::Evidence::OUTSTANDING_STATES & aasm_states).present?`